01_fib/nickolee/fibonacci_test.go

```diff
@@ -0,0 +1,53 @@
+package main
+
+import (
+	"bytes"
+	"reflect"
+	"strconv"
+	"testing"
+)
+
+func TestFibonacci(t *testing.T) {
+	var buf bytes.Buffer
+	out = &buf
+
+	main()
+
+	expected := strconv.Quote(`1
+1
+2
+3
+5
+8
+13
+`)
+	actual := strconv.Quote(buf.String())
+
+	if expected != actual {
+		t.Errorf("TestFibonacci: Unexpected output in main()")
+	}
+}
+
+func TestFibonacciUpperLimit(t *testing.T) {
+	var buf bytes.Buffer
+	out = &buf
+	fib(93)
+	expected := strconv.Quote("Overflow error. Please use an argument less than 92")
```

Comment:
any reason why you're using strconv.Quote()?

Reply:
Followed Julia's example in 00_hello_world/juliaogris/helloworld